hosting/digitalocean/files/var/lib/cloud/scripts/per-instance/001_onboot

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# go into the app dir
+cd /root
+
+# fetch envoy and docker-compose files
+wget https://raw.githubusercontent.com/Budibase/budibase/master/hosting/docker-compose.yaml
+wget https://raw.githubusercontent.com/Budibase/budibase/master/hosting/envoy.yaml
+wget https://raw.githubusercontent.com/Budibase/budibase/master/hosting/hosting.properties
+
+# Create .env file from hosting.properties using bash and then remove it
+while read line; do
+		uuid=$(uuidgen)
+		echo $line | sed "s/budibase/$uuid/g" | sed "s/testsecret/$uuid/g" >> .env
```

Comment:
What does the `testsecret` mean here?

Reply:
it's the default value for the `JWT_SECRET` value, so it just replaces it with a UUID.